# pkg_path
# pkg_path: "/home/lin/ros_code/calibration_ws/src/lidar2cam_calibration"

# Topics
topic_img: "/camera/color/image_raw"  
topic_cloud: "/velodyne_points"

topic_img_new: "/aruco_image"
topic_cloud_seg: "/points_seg"
topic_cloud_filter: "/points_filter"

# frame_id
frame_id: "velodyne"


# Dynamic rqt_reconfigure default bounds 点云的选取范围，用默认的就是，之后可以在线调
feature_extraction:
  x_min: 0.0
  x_max: 1.5
  y_min: -1
  y_max: 1
  z_min: -0.23
  z_max: 1.0

# Properties of chessboard calibration target  ,单位m
# ArUco marker是一个由二进制矩阵组成的正方形标记
ArUco marker:
  ArucoID: 8 #  Dictionary创建ArUco marker字典所需
  marker_size: .0.1  # ArUco marker边长 单位m
  min_detected_markers: 3  # 检测到ArUco marker最少3个，共4个

  board_size:   # ArUco 标定板的边长 单位m
    board_height: 0.7
    board_width: 0.5 
  
  
  marker_center_distance:  # ArUco marker标志图案的中心距
    delta_width_qr_center: 0.55
    delta_height_qr_center: 0.35

# Camera internal reference 相机内参,畸变
fx: 617.201
fy: 617.362
cx: 324.637
cy: 242.462

k1: 0.0
k2: 0.0
k3: 0.0
p1: 0.0
p2: 0.0

# optimization_mode优化方式,相机内参是否参数,默认0只优化T,1为同时优化K,T
optimization_mode: 0

